droid-command-line/pom.xml

```diff
@@ -46,10 +46,11 @@
                     <dependency>
                         <groupId>uk.gov.nationalarchives</groupId>
                         <artifactId>droid-build-tools</artifactId>
-                        <version>6.5-SNAPSHOT</version>
+                        <version>6.5-VITAM</version>
                     </dependency>
                 </dependencies>
             </plugin>
+<!--
```

Comment:
Why is the `maven-dependency-plugin` commented out now?

Reply:
Becaused it's managed at the general level droid-parent